torch_em/data/datasets/util.py

```diff
@@ -225,6 +250,33 @@ def add_instance_label_transform(
     return kwargs, label_dtype
 
 
+def update_kwargs_for_resize_trafo(kwargs, patch_shape, resize_inputs, resize_kwargs=None):
+    """
+    Checks for raw_transform and label_transform incoming values.
+    If yes, it will automatically merge these two transforms to apply them together.
+    """
+    if resize_inputs:
+        assert isinstance(resize_kwargs, dict)
+        patch_shape = None
+
+        raw_trafo = ResizeInputs(target_shape=resize_kwargs["patch_shape"], is_rgb=resize_kwargs["is_rgb"])
+        label_trafo = ResizeInputs(target_shape=resize_kwargs["patch_shape"], is_label=True)
+
+    if "raw_transform" in kwargs:
+        trafo = ConcatTransforms(transform1=kwargs["raw_transform"], transform2=raw_trafo)
+        kwargs["raw_transform"] = trafo
+    else:
+        kwargs["raw_transform"] = raw_trafo
```

Comment:
We should use the default raw transform to ensure data standardization here. See https://github.com/constantinpape/torch-em/blob/main/torch_em/segmentation.py#L258

Reply:
Thanks for the mention. I should double check as almost all the medical datasets are missing this. Will take care of these in the incoming PRs (and wherever needed).